kernel/kernel-defaults/src/test/scala/io/delta/kernel/defaults/DeltaTableSchemaEvolutionSuite.scala

```diff
@@ -88,16 +81,14 @@ class DeltaTableSchemaEvolutionSuite extends DeltaTableWriteSuiteBase with Colum
           TableConfig.COLUMN_MAPPING_MODE.getKey -> "id",
           TableConfig.ICEBERG_COMPAT_V2_ENABLED.getKey -> "true"))
 
-      val currentSchema = table.getLatestSnapshot(engine).getSchema()
+      val currentSchema = table.getLatestSnapshot(engine).getSchema
       val newSchema = new StructType()
         .add("a", StringType.STRING, true, currentSchema.get("a").getMetadata)
 
-      table.createTransactionBuilder(engine, testEngineInfo, Operation.MANUAL_UPDATE)
-        .withSchema(engine, newSchema)
-        .build(engine).commit(engine, emptyIterable())
+      updateTableMetadata(engine, tablePath, newSchema)
 
-      val structType = table.getLatestSnapshot(engine).getSchema
-      assertColumnMapping(structType.get("a"), 1)
+      val actualSchema = getMetadata(engine, tablePath).getSchema
+      assert(actualSchema == newSchema)
```

Comment:
These tests (that you updated) are mostly just checking that we don't (1) fail when the operation should succeed (2) validate the newSchema is correctly committed to the table right?

Reply:
Like these test cases updated don't include ones where Kernel is updating any column metadata. We are pre-populating the field ids and everything